tests/unit/test_api.js

```diff
@@ -12,7 +12,7 @@ const { Connection }			= require("../../src/index.js");
 
 describe("Javascript API", () => {
 
-    it("should call zome function", async () => {
+    it("should call COMB.call", async () => {
```

Comment:
Just noticed this test name was a little misleading. It doesn't test as much as that name implies.